syncstorage/tests/functional/test_storage.py

```diff
@@ -802,15 +807,16 @@ def test_batch(self):
         self.assertEquals(len(res['success']), max_count)
         self.assertEquals(len(res['failed']), 5)
 
-        # The test config has max_bytes=1M.
-        # Uploading 5 210MB items should produce one failure.
-        max_bytes = get_limit_config(self.config, 'max_post_bytes')
-        self.assertEquals(max_bytes, 1024 * 1024)
-        bsos = [{'id': str(i), 'payload': "X" * (210 * 1024)}
-                for i in range(5)]
+        # Uploading N+1 210kB items should produce one failure.
+        item_size = (210 * 1024)
+        max_items = max_bytes / item_size
+        self.assertTrue(max_items * item_size < max_bytes)
```

Comment:
this seems to just be checking |(max_bytes / item_size) * item_size < max_bytes| (and similarly below)

Reply:
Yes, the idea was to check that `item_size` is not an exact multiple of `max_bytes` but I don't think that's actually necessary, I'll remove it.